plugins/woocommerce/tests/e2e-pw/utils/api.js

```diff
@@ -13,6 +13,42 @@ if ( process.env.CONSUMER_KEY && process.env.CONSUMER_SECRET ) {
 	} );
 }
 
+/**
+ * Allow explicit construction of api client.
+ */
+const constructWith = ( consumerKey, consumerSecret ) => {
+	api = new wcApi( {
+		url: config.use.baseURL,
+		consumerKey,
+		consumerSecret,
+		version: 'wc/v3',
+	} );
+};
```

Comment:
I needed to use the `api` client within `global-setup.js` so I had to come up with a way to explicitly construct it.